app/initializers/export-application-global.js

```diff
@@ -1,7 +1,8 @@
 import Ember from 'ember';
 import config from '../config/environment';
 
-export function initialize(container, application) {
+export function initialize() {
+  var application = arguments[1] || arguments[0];
```

Comment:
> Are you saying that the initializers that are internal to Ember itself are triggering this deprecation?

Yes

> Can you open an issue for that

I'll open a PR. The particulars of the fix are likely to be simple enough that I don't mind having to re-do them after feedback